server/data/PunishmentResult.ts

```diff
@@ -196,7 +177,6 @@ export function flattenPunishment(punishment: PunishmentDataWithSchedule): Punis
     ...(endDate && { endDate }),
     ...(activatedFrom && { activatedFrom }),
     ...(consecutiveReportNumber && { consecutiveReportNumber }),
-    ...(consecutiveReportAvailable && { consecutiveReportAvailable }),
```

Comment:
This was not present on `PunishmentData` which allowed me to refactor
```
export function flattenPunishment(punishment: PunishmentDataWithSchedule): PunishmentData
```
to
```
export function flattenPunishment(punishment: PunishmentDataWithSchedule): PunishmentDataV2
```
The API isn't expecting it (see [PunishmentsController.PunishmentRequest](https://github.com/ministryofjustice/hmpps-manage-adjudications-api/blob/3e95d136b89f1ef586a6b5b32f40990c3b33f9a5/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/reported/PunishmentsController.kt#L40)) - so think we're ok here?

Reply:
Added that back in with https://github.com/ministryofjustice/hmpps-manage-adjudications/pull/760/commits/dac99d5d70524114bbbca096f1ade80a09a49865

`consecutiveReportAvailable` is used in a couple of places - we can look to refactor the use when we tackle NN-5370

Reply:
Fixed with https://github.com/ministryofjustice/hmpps-manage-adjudications/pull/761